pkg/component/autoscaling/vpa/vpa_test.go

```diff
@@ -1405,6 +1420,53 @@ var _ = Describe("VPA", func() {
 				managedResourceName = "vpa"
 			})
 
+			Context("Different ServiceMonitor labels", func() {
+				Context("when used with Garden cluster", func() {
+					BeforeEach(func() {
+						isGardenCluster := true
+						vpa = vpaFor(component.ClusterTypeSeed, isGardenCluster)
+						Expect(vpa.Deploy(ctx)).To(Succeed())
+					})
+
+					It("should label `admission-controller` with `prometheus=garden`", func() {
```

Comment:
```suggestion
					It("should label vpa-admission-controller ServiceMonitor with `prometheus=garden`", func() {
```

Reply:
Thanks for the suggested wording. You can find the fixed version in my latest updates.